src/Application.php

```diff
@@ -15,6 +15,7 @@ public function __construct(
         protected int $activityTimeout,
         protected array $allowedOrigins,
         protected int $maxMessageSize,
+        protected ?int $maxConnections = null,
         protected array $options = [],
```

Comment:
Is this not a BC break? It would have been better to insert after `$options`... 🤷🏼‍♂️